sdk/core/azure-core/test/ut/context.cpp

```diff
@@ -197,40 +165,21 @@ TEST(Context, Chain)
   auto c7 = c6.WithValue("c7", "7");
   auto finalContext = c7.WithValue("finalContext", "Final");
 
-  auto& valueT2 = finalContext["c2"];
```

Comment:
Add the `[]` operator instead of removing it

Reply:
See my comment above.  The template approach for the values results in the [] producing less than ideal code.  We could add it in the future and it wouldn't be breaking.